game/board.js

```diff
@@ -104,6 +105,35 @@ class Board {
     return null;
   }
 
+  loadGame() {
+    var savedGame = JSON.parse(localStorage.getItem('progress'));
+    console.log(savedGame);
+
+    if (typeof(savedGame.score) !== undefined) {
+      console.log(savedGame.score);
+      this.score = savedGame.score;
+    }
+
+    if (typeof(savedGame.game) !== undefined) {
+      this.tiles.forEach((tile) => tile.clear());
+      const transitions = this.tiles.map((tile) => new TileTransition(tile));
+      var game = savedGame.game;
+      // var game = savedGame.game.map((savedTile) => new Tile(savedTile));
+      const empty = transitions.filter((transition) => transition.isEmpty());
+      let i = 0;
+      for (const transition of empty) {
+        transition.loadTile(game[i]);
+        this.tiles[i].coordinates = game[i].coordinates;
+        this.tiles[i].value = transition.newValue;
+        i++;
+      }
+      return transitions;
+    } else {
+      return this.reset()
+    }
+    
+  }
+
```

Comment:
The above function loads the previous save in the form of transitions to the board once the page is reloaded